Tables/FE8 Nightmare modules/Quote editors/Death Quote Editor.csv

```diff
@@ -46,7 +46,7 @@ INLINE NewDeathQuoteTable,Character,##UNKNOWN##,Chapter,Event ID,##UNKNOWN##,Tex
 0x2c,0xbe,0xff00,0x16,0x2,0x0,0x0,0x0
 0x2d,0xbe,0xff00,0x23,0x2,0x0,0x0,0x0
 0x2e,0x1,0xff00,0xff,0x65,0x0,0xc23,0x0
-0x2f,Inferos,0xff00,0xff,0x2,0x0,0xc24,0x0
+0x2f,Inferos,0xff00,0xff,0x2,0x0,PlaceholderText,0x0
```

Comment:
This is where player character death quotes are, actually. Could you use one of the unused slots above Vedge's quote? (entry 0x2e)

Reply:
whenever I edit these tables I'm terrified I'll end up replacing someone else's thing